src/manager.cpp

```diff
@@ -257,18 +257,20 @@ bool Manager::addBookFromPath(const std::string& pathToOpen,
 }
 
 void Manager::addBooksFromDirectory(const std::string& path,
-                                    const bool skipInvalid,
                                     const bool verboseFlag)
 {
   std::set<std::string> iteratedDirs;
   std::queue<std::string> dirQueue;
   dirQueue.push(fs::absolute(path).u8string());
+  int totalBooksAdded = 0;
+  if (verboseFlag)
+    std::cout << "Starting BFS traversal at root directory: " << dirQueue.front() << std::endl;
```

Comment:
I would prefer if the message reads "Adding books from the directory tree: /path/to/dir"